cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -268,10 +268,7 @@ func createKubeConfigFileIfNotExists(outDir, filename string, config *clientcmda
 		}
 		fmt.Printf("[kubeconfig] Writing %q kubeconfig file\n", filename)
 		err = kubeconfigutil.WriteToDisk(kubeConfigFilePath, config)
-		if err != nil {
-			return errors.Wrapf(err, "failed to save kubeconfig file %q on disk", kubeConfigFilePath)
-		}
-		return nil
+		return errors.Wrapf(err, "failed to save kubeconfig file %q on disk", kubeConfigFilePath)
```

Comment:
this is not correct because it always returns a wrapped error instead of nil?

Reply:
get it, thank you

Reply:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/55286565/181742784-5eba7d52-a2c5-439b-9847-612a4df30891.png">
I think it should be okay.

Reply:
ok seems good